src/Npgsql/NpgsqlCommand.cs

```diff
@@ -1143,16 +1143,24 @@ async ValueTask<NpgsqlDataReader> ExecuteReaderAsync(CommandBehavior behavior, b
                         if (connector.Settings.MaxAutoPrepare > 0)
                         {
                             var numPrepared = 0;
+
+                            var now = DateTime.UtcNow;
```

Comment:
Could you add a comment why is it important to store the current timestamp in a local variable?

Reply:
added